tests/test_platform.py

```diff
@@ -41,4 +44,939 @@ def test_platform_create_bad_platform_empty_fields():
 def test_platform_create_noop():
     # type: () -> None
     existing = Platform.create("linux-x86_64-cp-27-mu")
-    assert Platform.create(existing) == existing
+    assert Platform.create(existing) is existing
+
+
+def test_platform_supported_tags():
+    platform = Platform.create("macosx-10.13-x86_64-cp-36-m")
+
+    # A golden file test. This could break if we upgrade Pip and it upgrades packaging which, from
+    # time to time, corrects omissions in tag sets.
+    assert (
+        tuple(
+            itertools.chain.from_iterable(
+                tags.parse_tag(tag)
+                for tag in (
+                    "cp36-cp36m-macosx_10_13_x86_64",
```

Comment:
Could you please save this to something like a .txt file and load as a fixture? I think it will keep this test file much easier to work with and to read.

Reply:
Yeah, sg.